imap_processing/cdf/config/imap_codice_global_cdf_attrs.yaml

```
@@ -1,61 +1,67 @@
-  characterizes solar wind ions including the mass and composition of highly energized particles (called suprathermal)
-  from the Sun. CoDICE combines an electrostatic analyzer(ESA) with a Time-Of-Flight versus Energy (TOF / E) subsystem
-  to simultaneously measure the velocity, arrival direction, ionic charge state, and mass of specific species of ions in
-  the LISM. CoDICE also has a path for higher energy particles to skip the ESA but still get measured by the common
-  TOF / E system. These measurements are critical in determining the Local Interstellar Medium (LISM) composition and
-  flow properties, the origin of the enigmatic suprathermal tails on the solar wind distributions and advance
-  understanding of the acceleration of particles in the heliosphere.
+instrument_base: &instrument_base
+  Descriptor: &desc
+    CoDICE>Compact Dual Ion Composition Experiment
+  TEXT: &txt >
+    The Compact Dual Ion Composition Experiment (CoDICE) will measure the distributions and composition of interstellar
+    pickup ions (PUIs), particles that make it through the heliosheath into the heliosphere. CoDICE also collects and
+    characterizes solar wind ions including the mass and composition of highly energized particles (called suprathermal)
+    from the Sun. CoDICE combines an electrostatic analyzer(ESA) with a Time-Of-Flight versus Energy (TOF / E) subsystem
+    to simultaneously measure the velocity, arrival direction, ionic charge state, and mass of specific species of ions in
+    the LISM. CoDICE also has a path for higher energy particles to skip the ESA but still get measured by the common
+    TOF / E system. These measurements are critical in determining the Local Interstellar Medium (LISM) composition and
+    flow properties, the origin of the enigmatic suprathermal tails on the solar wind distributions and advance
+    understanding of the acceleration of particles in the heliosphere.
 
 imap_codice_l1a_hskp:
-    Descriptor: *desc
-    TEXT: *txt
+    <<: *instrument_base
     Data_type: L1A_hskp->Level-1A Housekeeping Data
     Logical_source: imap_codice_l1a_hskp
     Logical_source_description: IMAP Mission CoDICE Instrument Level-1A Housekeeping Data.
 
+imap_codice_l1a_lo_counters_aggregated:
+    <<: *instrument_base
+    Data_type: L1A_lo-counters-aggregated->Level-1A Lo Aggregated Instrument Counts Data
+    Logical_source: imap_codice_l1a_lo-counters-aggregated
+    Logical_source_description: IMAP Mission CoDICE Lo Level-1A Aggregated Instrument Counts Data.
+
+imap_codice_l1a_lo_counters_singles:
+    <<: *instrument_base
+    Data_type: L1A_lo-counters-aggregated->Level-1A Lo Single Instrument Counts Data
+    Logical_source: imap_codice_l1a_lo-counters-singles
+    Logical_source_description: IMAP Mission CoDICE Lo Level-1A Single Instrument Counts Data.
+
 imap_codice_l1a_lo_sw_angular_counts:
-    Descriptor: *desc
-    TEXT: *txt
+    <<: *instrument_base
```

Can you add `Data_level` to remaining of these products? As noted in my PR, without this, it will produce invalid CDF.
```suggestion
    <<: *instrument_base
    Data_level: 1A
```